maintenance_stock/models/stock_warehouse.py

```diff
@@ -64,8 +67,8 @@ def _get_sequence_values(self):
             },
         }
 
-    def _get_locations_values(self, vals):
-        sub_locations = super()._get_locations_values(vals)
+    def _get_locations_values(self, vals, code=False):
+        sub_locations = super()._get_locations_values(vals, code)
         code = vals.get("code") or self.code
```

Comment:
Is it this the same code provided by the inherited function?

Also, you are overriding a parameter of the function... you should review it.

Reply:
@etobella you're right, that code was discussed in v13 here: https://github.com/OCA/maintenance/pull/337#pullrequestreview-1509522122

I've noticed two things:
- v13 recommendation wasn't finally attended (`code` method parameter was finally unused), not fault of this PR.
- commit history is definetely wrong, because this change shouldn't appear here.

@Reyes4711-S73 please rebuild commit history. See e.g.https://github.com/OCA/maintenance/commit/a7fa359d34dc7c6aa79a65b47f15c79a84a8e30a#diff-6f3891aa1e983a123b23df8d101c909e1d2e6812915e7dbda2a820400a25c8b0 , that commit does not belong to v13 implementation and have merge conflicts syntax indeed. You should start again the migration process and strictly add needed changes in last `Migration to 14.0` commit.

@etobella sorry I didn't notice that problem...